themes/palenight-italic.json

```diff
@@ -1181,53 +1181,97 @@
       }
     },
     {
-      "name": "Markdown Headings",
-      "scope": "markup.heading.markdown",
+      "name": "Markup Headings",
+      "scope": "markup.heading",
       "settings": {
         "foreground": "#82b1ff"
       }
     },
     {
-      "name": "Markdown Italics",
-      "scope": "markup.italic.markdown",
+      "name": "Markup Italics",
+      "scope": "markup.italic",
       "settings": {
         "foreground": "#c792ea",
         "fontStyle": "italic"
       }
     },
     {
-      "name": "Markdown Bold",
-      "scope": "markup.bold.markdown",
+      "name": "Markup Bold",
+      "scope": "markup.bold",
       "settings": {
         "foreground": "#ffcb6b",
         "fontStyle": "bold"
       }
     },
     {
-      "name": "Markdown Quote + others",
-      "scope": "markup.quote.markdown",
+      "name": "Markup Quote + others",
+      "scope": "markup.quote",
       "settings": {
         "foreground": "#697098",
         "fontStyle": "italic"
       }
     },
     {
-      "name": "Markdown Raw Code + others",
-      "scope": "markup.inline.raw.markdown",
+      "name": "Markup Raw Code + others",
+      "scope": "markup.inline.raw",
       "settings": {
         "foreground": "#80CBC4"
       }
     },
     {
-      "name": "Markdown Links",
+      "name": "Markup Links",
       "scope": [
-        "markup.underline.link.markdown",
-        "markup.underline.link.image.markdown"
+        "markup.underline.link",
+        "markup.underline.link.image"
       ],
       "settings": {
         "foreground": "#ff869a"
       }
     },
+    {
+      "name": "Markup Fenced Code",
+      "scope": [
+        "fenced_code.block.language"
```

Comment:
Can you share a snippet of code that contains this token?

Reply:
In fact, the asciidoc vscode extension doesn't provides this scope yet, I thought it would be a good addition to generic markup languages (and I personnaly configured it for my markdown files). I'm trying to find examples, but I don't find any other vscode extension supporting this beside Markdown, so it may have been a different Pull Request for markdown I guess:

~~~markdown
```python
print("hello")
```
~~~

Reply:
I see, makes sense.